frontend/src/css/imports.css

```diff
@@ -6743,18 +6744,18 @@ body,
   position: relative;
   height: calc(100% - 20px - 25px);
 }
-#worksheet .editable-field:hover {
+.editable-field:hover {
   cursor: text;
-  border-radius: 4px;
-  box-shadow: 1px 1px 1px 1px gray;
+  border: 2px solid #eeeeee;
+  border-radius: 2px;
 }
 #ws_search {
   position: fixed;
   width: 100%;
   margin: 0;
   padding: 10px 0;
   text-align: center;
-  background-color: #F8F9FA;
+  background: rgba(245, 245, 245, 0.55);
```

Comment:
Can you make this a bit more opaque? Right now it's too transparent and a bit hard to read:

![image](https://user-images.githubusercontent.com/1689183/67315388-5003d180-f4bb-11e9-856a-b41a8d4c3baa.png)